storage/postgres/migrations/20200114101000_dashboard_url.up.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+ALTER TABLE service_instances ADD COLUMN dashboard_url varchar(100);
```

Comment:
Not sure that 100 will be enough for all services. Check whether OSB spec says something. At first glance it does not say anything about length as far as I checked

Reply:
It's 16000 chars in CC DB, I can make it the same.